t/venom/test_suites/wired_dot1x_eap_tls_pki/36_restart_radius_services.yml

```diff
@@ -0,0 +1,9 @@
+name: Restart RADIUS services
+testcases:
+- name: restart_radius_services
+  steps:
+  - type: pf_api_service_restart
+    service: radiusd-auth
+
+  - type: pf_api_service_restart
+    service: pfacct
```

Comment:
Not necessary because we restart RADIUS only to take new RADIUS certificats into account